doc-content/enterprise-only/getting-started-kogito/proc-kogito-designing-app-drl-unit.adoc

```diff
@@ -0,0 +1,53 @@
+[id="proc-kogito-designing-app-drl-unit_{context}"]
+= Using DRL rule units as an alternative decision service
+
+As an alternative to using Decision Model and Notation (DMN) to define this example decision service, you can also use a Drools Rule Language (DRL) file implemented as a rule unit.
+
+A DRL rule unit is a module for rules and a unit of execution. A rule unit collects a set of rules with the declaration of the type of facts that the rules act on. A rule unit also serves as a unique namespace for each group of rules. A single rule base can contain multiple rule units. You typically store all the rules for a unit in the same file as the unit declaration so that the unit is self-contained. For more information about rule units, see {URL_DEVELOPING_DECISION_SERVICES}#assembly-drl-rules[_{DRL_RULES}_].
+
+.Procedure
+. In the `src/main/resources` folder of your example project, instead of using a DMN file, add the following `PersonRules.drl` file:
++
+--
+.Example `PersonRules` DRL file
```

Comment:
Differently from DMN, this DRL file is not sufficient to have an automatically generated microservice exposing a REST endpoint. I'm not sure if this was the intent of this section, so please confirm if this is ok.

Reply:
Added the following query to the drl file:
query persons
  $p : /person[ adult ]